server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedIT.java

```diff
@@ -496,7 +496,7 @@ public void testExplainWithSingleDoc() throws Exception {
         assertThat(searchResponse.getHits().getTotalHits().value(), equalTo(1L));
         Explanation explanation = searchResponse.getHits().getHits()[0].getExplanation();
         assertThat(explanation.getValue(), equalTo(searchResponse.getHits().getHits()[0].getScore()));
-        assertThat(explanation.toString(), startsWith("0.36464313 = Score based on 2 child docs in range from 0 to 1"));
+        assertThat(explanation.toString(), startsWith("0.16574687 = Score based on 2 child docs in range from 0 to 1"));
```

Comment:
Yay! This number reflects what I expected.

Scores should be scaled by a factor of `(1/k1+1) = 1 / 2.2`. Sure enough, `2.2 * 0.16574687 = .364643114` (close enough).